lib/nexpose/password_policy.rb

```diff
@@ -0,0 +1,58 @@
+module Nexpose
+  require 'json'
+  # Configuration structure for password policies.
+  class PasswordPolicy < APIObject
+
+    attr_accessor :policyName
+    attr_accessor :minLength
+    attr_accessor :maxLength
+    attr_accessor :capitals
+    attr_accessor :digits
+    attr_accessor :specialChars
+
+    def initialize(policyName:, minLength:, maxLength:, specialChars:, capitals:, digits:)
```

Comment:
I'd leave `#to_h` since it is used for the `#to_json` conversion. We'd want to use the same keys there.